.travis.yml

```diff
@@ -3,30 +3,76 @@ sudo: true
 language: go
 
 go:
-  - 1.9
+  - 1.10.x
+  - 1.11.x
   - master
 
+dist: trusty
+addons:
+  apt:
+    sources:
+      - ubuntu-toolchain-r-test
+      - llvm-toolchain-trusty
+    packages:
+      - g++-4.9
+      - clang-8
+      - clang-tools-8
+      - clang-8-doc
+      - libclang-common-8-dev
+      - libclang-8-dev
+      - libclang1-8
+      - libllvm8
+      - llvm-8
+      - llvm-8-dev
+      - llvm-8-runtime
+      - libfuzzer-8-dev
+      - lldb-8
+      - lld-8
+env:
+  - CC=clang-8 CGO_CXXFLAGS=-std=c++11 CGO_LDFLAGS_ALLOW='Wl,(search_paths_first|headerpad_max_install_names)' CGO_CPPFLAGS="`llvm-config-8 --cppflags`" CGO_LDFLAGS="`llvm-config-8 --ldflags --libs --system-libs all`"
+
 # Skip install step. Control by ourself
 install: true
 
 matrix:
   allow_failures:
     - go: master
-  # This mean if stable go version pass the test, will mark test green
   fast_finish: true
-
-services:
-  - docker
+  include:
+    - go: 1.10.x
```

Comment:
I think this can be removed.

Reply:
Because we would have llvm 8 by default, else is something defined in include block